### PR TITLE
chore(prettier): improve ci

### PR DESCRIPTION
**Pull request by stavares843**
## What kind of change does this PR introduce?

Improving Prettier CI

## What is the current behavior?

Atm, there's no annotations when prettier has any suggestions.

## What is the new behavior?

Adding this, CI will add annotations that the author can or not commit directly from the bot.

## Additional context

Captura de ecrã 2022-10-28, às 22 18 37

### CODE REVIEW

**Comment by stavares843** [comment]
cc @kiwicopple let me know your thoughts, as you were the one updating this CI the most 🔨

**Comment by pucho** [comment]
@stavares843 Do you mind giving a better explanation on what this does? I don't follow what this solves exactly, thanks!

**Comment by stavares843** [comment]
it adds annotations like the ones in the screenshot in the description when running the linters in CI 🔨 it makes the feedback from CI more visible and you can address it directly from the annotation as well

let me know if that helps!

**Comment by pucho** [comment]
@sweatybridge Do you mind taking a look at the github action, thanks!

File: .github/workflows/prettier.yml [hunk 1]
```diff
@@ -13,8 +13,13 @@ jobs:
...
       # disable this check while we migrate to monorepo
       - name: Run prettier
-        run: |-
-          npx prettier -c 'apps/**/*.{js,jsx,ts,tsx,css,md,json}' 
+        uses: EPMatt/reviewdog-action-prettier@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
+          level: warning
+          command: npx prettier -c 'i18n/**/*.{js,jsx,ts,tsx,css,md,json}'
```
Comment (sweatybridge) [inline .github/workflows/prettier.yml:21]:
```suggestion
          command: npx prettier -c 'apps/**/*.{js,jsx,ts,tsx,css,md,json}'
```

**Review by sweatybridge — APPROVED** [review]
LGTM. Let's try it out

**Comment by stavares843** [comment]
here's a example of the bot working 🔨 

https://github.com/supabase/supabase/pull/11353/files



Captura de ecrã 2022-12-30, às 22 15 05

**Pull request merged**